src/popup/router/pages/Tips.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="tips">
+    <AccountInfo />
+    <BalanceInfo />
+    <Tabs>
+      <RouterLink
+        :to="{ name: 'tips-send' }"
+        exact-path
+      >
+        <TipIcon /> {{ $t('pages.titles.send-tip') }}
+      </RouterLink>
+      <RouterLink :to="{ name: 'tips-claim' }">
+        <ClaimTipsIcon /> {{ $t('pages.titles.claim-tips') }}
```

Comment:
Icon seems to be different from designs in Figma, as well as letter casing:

![Screenshot 2021-04-26 at 11 29 36](https://user-images.githubusercontent.com/482351/116052995-e00f7200-a682-11eb-938a-cb87599d7d90.png)